doc/integrator/extend_data_model.rst

```diff
@@ -1,6 +1,10 @@
 Extend the data model
 ---------------------
 
+.. note::
+
+    This is not possible in the simple application mode
```

Comment:
Same as above, let's be more explicit.
"Extending the data model is not possible in the simple application mode."